PowerEditor/installer/nativeLang/croatian.xml

```diff
@@ -1067,6 +1067,7 @@ Vidi Notepad++ online priručnik (Preferences)."/>
                     <Item id="6903" name="Dijalog pretraživanja ostaje otvoren nakon pretraživanja koje otvara prozor s rezultatima"/>
                     <Item id="6904" name="Potvrdi Zamijeni sve u svim otvorenim dokumentima"/>
                     <Item id="6905" name="Zamijeni: Ne prelazi na sljedeće pojavljivanje"/>
+                    <Item id="6906" name="Prozor rezultata pretraživanja: prikaži samo jedan rezultat po pronađenom retku (ne primjenjuje se na RTL način)"/>
```

Comment:
"(not applied to RTL mode)" has been removed in this entry, see: https://github.com/notepad-plus-plus/notepad-plus-plus/commit/059bca3315dbd71fbf235c8bc87eaacbd87a3cec
Sorry for the inconvenience.